### PR TITLE
Add release instructions for opentelemetry proto.

### PR DESCRIPTION
**Pull request by bogdandrutu**
Once this is merge I will follow the instructions and create the release branch + the tag.

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+$ git checkout -b cherrypick v$MAJOR.$MINOR.x
+$ git cherry-pick -x $COMMIT
+$ git commit -a -m "Cherry-pick commit $COMMIT"
+```
+
+- Go through PR review and merge it to GitHub v$MAJOR.$MINOR.x branch.
+
+- Tag a new patch release when all commits are merged.
+
+## Announcement
```
Comment (SergeyKanzhelev) [inline RELEASING.md]: it will be nice to start tracking `CHANGELOG.md` in this repository

Comment (bogdandrutu) [reply RELEASING.md]: Done.

File: RELEASING.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+## Update release versions in documentations and CHANGELOG files
+
+After releasing is done, you need to update
+[README.md](https://github.com/open-telemetry/opentelemetry-proto/blob/master/README.md) and
+[CHANGELOG.md](https://github.com/open-telemetry/opentelemetry-proto/blob/master/CHANGELOG.md).
+
+Create a PR to mark the new release in
+[CHANGELOG.md](https://github.com/census-instrumentation/opencensus-java/blob/master/CHANGELOG.md)
+on master branch.
```
Comment (arminru) [inline RELEASING.md]: I'd prefer the changes to be added to the changelog _before_ merging. This way, the changelog on each branch would be up to date and outline the changes introduced by that version.
Ideally, each PR to master would add a one line summary of its changes to the changelog right away, if they are noteworthy.

Comment (bogdandrutu) [reply RELEASING.md]: this item says to just move the entries from unreleased to the version that released the changes, I do agree that each significant PR should add an entry to the unreleased section.

Comment (bogdandrutu) [reply RELEASING.md]: Also we can tune this on the way :) Let's start with something to unblock this. :)

**Pull request merged**